bin/plugin/commands/packages.js

```diff
@@ -237,8 +239,46 @@ async function updatePackages(
 		true,
 		abortMessage
 	);
-	git.commit( gitWorkingDirectoryPath, 'Update changelog files', [ './*' ] );
+	const commitHash = await git.commit(
+		gitWorkingDirectoryPath,
+		'Update changelog files',
+		[ './*' ]
+	);
 	log( '>> Changelog files changes have been committed successfully.' );
+
+	return commitHash;
+}
+
+/**
+ * Publishes the new packages to npm
+ *
+ * @param {string} gitWorkingDirectoryPath Git working directory path.
+ * @param {string} minimumVersionBump      Minimum version bump for the packages.
+ * @param {string} abortMessage            Abort Message.
+ */
+async function publishPackages(
+	gitWorkingDirectoryPath,
+	minimumVersionBump,
+	abortMessage
+) {
+	await runStep( 'Publishing the npm packages', abortMessage, async () => {
+		await askForConfirmation(
```

Comment:
We also need to run `npm whoami` to ensure that the person doing release is logged in to npm. Otherwise, the process will fail after new commit and release tags are created locally.